addons/web/static/tests/core/record_selectors/multi_record_selector.test.js

```diff
@@ -239,3 +264,30 @@ test("Backspace do nothing when the input is currently edited", async () => {
     await animationFrame();
     expect(".o_tag").toHaveCount(2);
 });
+
+// Desktop only because .o_data_row is only available on desktop
+test.tags("destop")("Can pass domain to search more", async () => {
```

Comment:
```suggestion
// Desktop only because a kanban view is used instead of a list in mobile
test.tags("desktop")("Can pass domain to search more", async () => {
```
typo in the tag lol